src/pacer.js

```diff
@@ -151,6 +151,18 @@ let PACER = {
     return /iquery.pl/.test(url) && !/[?&]/.test(url)
   },
 
+  // Returns true if the page contains a Transaction Receipt table
+  hasTransactionReceipt(){
+    return !!$('th>font:contains("Transaction Receipt")').length;
+  },
+
+  // Returns true if the description in the transaction receipt is "Search" and
+  // the title of the page is "Select a Case"
+  isCaseQueryAdvance: function(){
+    let title = document.querySelector('#cmecfMainContent>h2')
+    return !!$('td>font:contains(Search)').length && /Select a Case/i.test(title.innerHTML);
+  },
```

Comment:
If you do this, you can include the comment about when people search for first/middle/last *here* instead of up above, so that whenever people use isCaseQueryAdvance, they get that explanation of why we're using a regex to look for "Select a Case".

Reply:
You're right! I will include a check for the **transaction receipt table** inside the `isCaseQueryAdvance`.

I think we can improve the `isCaseQueryAdvance` method if we include the validations I listed in my other comment so we don't have to only rely on the receipt table.